src/modules/crafting.haml

```diff
@@ -0,0 +1,354 @@
+                                    A custom shapeless recipe node.
+                                    %a.left-ref-link{:href=>"#shapeless"} <i class="fa fa-chevron-down"></i>
+                                %td
+                                    %span.label.label-default Shapeless Recipe Sub-elements
+                            %tr
+                                %td
+                                    %pre
+                                        %code= html_escape('<smelt>')
+                                %td
+                                    A custom smelting recipe.
+                                    %a.left-ref-link{:href=>"#smelt"} <i class="fa fa-chevron-down"></i>
+                                %td
+                                    %span.label.label-default Smelt Recipe Sub-elements
+                            %tr
+                                %td
+                                    %pre
+                                        %code= html_escape('<disable>')
+                                %td Disable all recipes for this item.
+                                %td
+                                    %a{:href => "/reference/inventory#material_finder"} Material Name
+
+                %h5 Recipe Attributes
+                .table-responsive
+                    %table.table.table-striped.table-condensed
+                        %thead
+                            %tr
+                                %th Attribute
+                                %th Description
+                                %th Value
+                        %tbody
+                            %tr
+                                %td
+                                    %code override
+                                %td
+                                    Override all vanilla recipes resulting in this material and data.
+                                %td
+                                    %span.label.label-primary true/false
+                                %td false
+                            %tr
+                                %td
+                                    %code override-all
+                                %td
+                                    Override all vanilla recipes resulting in this material.
+                                %td
+                                    %span.label.label-primary true/false
+                                %td false
+
+                %h5 Recipe Sub-elements
+                .table-responsive
+                    %table.table.table-striped.table-condensed
+                        %thead
+                            %tr
+                                %th Element
+                                %th Description
+                                %th Value/Children
+                        %tbody
+                            %tr
+                                %td
+                                    %pre
+                                        %code= html_escape('<result>')
```

Comment:
This uses the same format as `<item>` in kits, so we should use the `material="..."` attribute and mention that all kit item features are supported.